ztp/source-crs/ClusterLogForwarder.yaml

```diff
@@ -9,25 +9,33 @@ spec:
 #  outputs: $outputs
 #  pipelines: $pipelines
 
-#apiVersion: "logging.openshift.io/v1"
+#apiVersion: "observability.openshift.io/v1"
 #kind: ClusterLogForwarder
 #metadata:
 #  name: instance
 #  namespace: openshift-logging
 #spec:
+#  serviceAccount:
+#   name: logcollector
```

Comment:
We should explicitly include this part since we have the service accounts defined.

Reply:
I've added the service account to the defined section